deploy/modules/statistics/statistics.tpl

```diff
@@ -1,36 +1,23 @@
 #!/bin/bash -x
 set -e
 
-statistics_bucket_name="04274532-55f0-11ed-bdc3-0242ac120002"
+if [[ "$(aws --version)" == *"aws-cli"* ]]
+then
+  now=$(date)
+  me=$(whoami)
+  example_path=$(pwd)
 
-now=$(date)
-me=$(whoami)
-file_name=$me.txt
-example_path=$(pwd)
+  file_name=$me-${salt}.gitignore.txt
```

Comment:
salt is only generated once and therefore only one metrics file will be present if multiple runs are invoked

Reply:
Yes but if you run terraform deploy multiple times at the same machine with the same Terraform state, the stats file will contains multiple rows

Reply:
I see, in that case, your file wont be JSON. It might make Linda harder life to process it with Athena. In that case, we cant change it.